sonar-scanner-engine/src/main/java/org/sonar/batch/source/DefaultSymbolizable.java

```diff
@@ -41,6 +41,11 @@ public Symbol newSymbol(int fromOffset, int toOffset) {
     }
 
     @Override
+    public Symbol newSymbol(int startLine, int startLineOffset, int endLine, int endLineOffset) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Symbol"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)